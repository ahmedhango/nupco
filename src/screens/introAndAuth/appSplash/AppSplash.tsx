import React from 'react';
import {Image} from 'react-native';
import {ScreenContainer} from '../../../components';
import {useAppSplash} from '../../../hooks/introAndAuth/useAppSplash';
import {styles} from './styles';

const AppSplash = (): JSX.Element => {
  useAppSplash();

  return (
    <ScreenContainer
      style={{
        ...styles.container,
      }}>
      <Image
        style={styles.loaderImg}
        source={require('./../../../assets/images/logo.png')}
      />
    </ScreenContainer>
  );
};

export default AppSplash;
