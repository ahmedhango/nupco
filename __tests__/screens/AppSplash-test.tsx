/**
 * @format
 */

import 'react-native';
import React from 'react';

import {render} from '@testing-library/react-native';

import AppSplash from '../../src/screens/introAndAuth/appSplash/AppSplash';

describe('AppSplach', () => {
  it('renders appSplash using Snapshots', () => {
    const appSplash = render(<AppSplash />);

    expect(appSplash.toJSON()).toMatchSnapshot();
  });
});
