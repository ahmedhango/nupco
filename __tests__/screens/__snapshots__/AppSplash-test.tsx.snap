// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppSplach renders appSplash using Snapshots 1`] = `
<View
  style={
    {
      "flex": 1,
      "height": "100%",
      "width": "100%",
    }
  }
>
  <RCTSafeAreaView
    style={
      {
        "backgroundColor": "#245584",
      }
    }
  />
  <View
    style={
      {
        "alignItems": "center",
        "backgroundColor": "#F2EDDC",
        "flex": 1,
        "justifyContent": "center",
        "paddingBottom": 33.76,
        "paddingHorizontal": 11.7,
        "width": "100%",
      }
    }
  >
    <Image
      source={
        {
          "default": "test-file-stub",
        }
      }
      style={
        {
          "borderRadius": 15,
          "height": 90,
          "resizeMode": "cover",
          "width": 90,
        }
      }
    />
  </View>
</View>
`;
